ios/Sources/MaaSTheme/Kotlin.swift

```diff
@@ -1,14 +1,19 @@
 import UIKit
 import SwiftUI
+import MaasCore
 
 @dynamicMemberLookup
 public struct Kotlin<T> {
 
-    let value: T
+    public let value: T
     public init(_ value: T) { self.value = value }
 
-    public subscript(dynamicMember keyPath: KeyPath<T, UIFont>) -> Font {
-        Font(value[keyPath: keyPath])
+    public subscript<V>(dynamicMember keyPath: KeyPath<T, V>) -> V {
+        value[keyPath: keyPath]
```

Comment:
This is to allow pass-through of values that don't need mapping?

Reply:
Pretty much. Also, AFAIK, to allow optionality.

Reply:
Nope. there's one more subscript below 🙈